sdk/standbypool/Azure.ResourceManager.StandbyPool/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
When do you want to release the SDK? Please specify the date by replacing the `Unreleased` with an expected data, the format is `yyyy-mm-dd`.

Reply:
Thanks for reminding, updated.